### PR TITLE
fix account_show + add --help wrap() + slight updates to --help order

### PR DESCRIPTION
**Pull request by elv-todd**
removed in 

commit 0df8289429c5f9c94235c79d86a3828c0b961a51
Date:   Fri Jul 8 10:47:03 2022 -0700

and better help width and command order

### CODE REVIEW

File: src/EluvioLive.js [hunk 1]
```diff
@@ -87,6 +87,23 @@ class EluvioLive {
...
 
+  /**
+   * Show info about this account.
+   */
+  async AccountShow() {
+    if (!this.client) {
+      throw Error("EluvioLive not intialized");
+    }
+
+    let tenantId = await this.client.userProfileClient.TenantId();
```
Comment (elv-stav) [inline src/EluvioLive.js:98]: Would this benefit from running these in parallel?
```js
const [tenantId, walletAddress, userWalletObject, userMetadata] = await Promise.all([
    this.client.userProfileClient.TenantId(),
    this.client.userProfileClient.WalletAddress(),
    this.client.userProfileClient.UserWalletObjectInfo(),
    this.client.userProfileClient.UserMetadata()
]);
```

Comment (elv-todd) [reply src/EluvioLive.js:98]: lgtm!  adding

File: utilities/EluvioLiveCli.js [hunk 2]
```diff
@@ -3342,6 +3342,7 @@ yargs(hideBin(process.argv))
   )
 
   .strict()
+  .wrap(yargs().terminalWidth() != null ? Math.min(120, yargs().terminalWidth()) : 120)
```
Comment (elv-todd) [inline utilities/EluvioLiveCli.js:3345]: need the null check to deal w/ `elv-live | less`, and the min just feels better for extremely wide terminals

**Comment by elv-todd** [comment]
```
$ ./elv-live
EluvioLive CLI

Usage: elv-live <command>

Commands:
  tenant_auth_token <path_or_body>                              Generate a tenant token
  nft_set_proxy <addr> [proxy_addr]                             Set a proxy on an NFT contract
  nft_balance_of <addr> <owner>                                 Call NFT ownerOf - determine if this is an owner
  nft_show <addr> [options]                                     Show info on this NFT
  nft_refresh <tenant> <addr>                                   Synchronize backend listings with fabric metadata for a
                                                                specific tenant's NFT. Requires tenant Key.
  nft_transfer <addr> <token_id> <to_addr> [options]            Transfer the specified NFT as the token owner
  transfer_errors <tenant>                                      Show tenant transfer failures. Used to identify payments
                                                                collected on failed transfers.
  nft_proxy_transfer <addr> <token_id> <from_addr> <to_addr>    Tranfer NFT as a proxy owner
  nft_build <library> <object> [options]                        Build the public/nft section based on asset metadata. If
                                                                --nft_dir is specified, will build a generative nft
                                                                based on *.json files inside the dir. See README.md for
                                                                more details.
  nft_pack_get_dist <hash>                                      Compute pack distribution based on NFT Template spec.
                                                                Saves to file.
  nft_pack_set_dist <hash>                                      Set the pack distribution for an NFT Template. Reads
                                                                distribution from file.
  nft_burn <addr> <token_id>                                    Burn the specified NFT as the owner
  nft_proxy_burn <addr> <token_id>                              Burn the specified NFT
  nft_set_transfer_fee <addr> <fee>                             Decode and look up a local NFT by external token ID
  nft_get_transfer_fee <addr>                                   Decode and look up a local NFT by external token ID
  nft_add_contract <tenant> <object> <cap> <name> <symbol>      Add a new or existing NFT contract to an NFT Template
  [options]                                                     object
  nft_add_offer <addr>                                          Add a redeemable offer to the NFT contract as the
                                                                contract owner or minter
  nft_remove_offer <addr> <offer_id>                            Remove (disable) a redeemable offer from the NFT
                                                                contract as the contract owner or minter
  nft_offer_redeemed <addr> <token_id> <offer_id>               Returns true if offer is redeemed
  nft_offer_active <addr> <offer_id>                            Returns true if offer is active
  nft_redeem_offer <addr> <redeemer> <token_id> <offer_id>      Redeem an nft offer
  as_nft_redeem_offer <addr> <tenant> <mint_helper_addr>        Redeem an nft offer using the authority service
  <token_id> <offer_id>
  nft_get_policy_permissions <object>                           Gets the policy and permissions of a content object.
  nft_set_policy_permissions <object> <policy_path> <addrs..>   Sets the policy and permissions granting NFT owners
                                                                access to a content object. When no addresses are
                                                                specified, only the policy is set.
  tenant_show <tenant> [options]                                Show info on this tenant
  tenant_set_token_uri <request_type> <tenant>                  Reset the token URI(s) for tenant NFT contract(s)
  <contract_address> <new_token_uri> [options]
  tenant_update_token_uri <tenant> <addr> <hash> [options]      Reset the token URI(s) for tenant NFT contract(s)
  tenant_balance_of <tenant> <owner>                            Show NFTs owned by this owner in this tenant using
                                                                contracts.
  fabric_tenant_balance_of <object> <owner> [options]           Show NFTs owned by this owner in this tenant by using
                                                                the Fabric EluvioLive object tree.
  site_show <library> <object>                                  Show info on this site/event
  site_set_drop <library> <object> <uuid> <start_date>          Set drop dates for a site/event
  [options]
  tenant_tickets_generate <tenant> <otp>                        Generate tickets for a given tenant OTP ID
  tenant_mint <tenant> <marketplace> <sku> <addr>               Mint a marketplace NFT by SKU as tenant admin
  tenant_primary_sales <tenant> <marketplace>                   Show tenant primary sales history
  tenant_secondary_sales <tenant>                               Show tenant secondary sales history
  tenant_sales <tenant>                                         Show tenant sales history
  tenant_wallets <tenant> [options]                             Show the wallets associated with this tenant
  tenant_nft_remove <tenant> <addr>                             Removes the nft address from the tenant contract
  tenant_nft_list <tenant>                                      List all tenant_nfts within a tenant contract
  tenant_has_nft <tenant> <addr>                                Searches tenant_nfts list in tenant contract and returns
                                                                true if exists
  tenant_add_consumers <group_id> [addrs..]                     Adds address(es) to the tenant's consumer group
  tenant_remove_consumer <group_id> <addr>                      Removes consumer from tenant consumer group
  tenant_has_consumer <group_id> <addr>                         Returns true or false if addr is in the tenant consumer
                                                                group
  marketplace_add_item <marketplace> <object> <name> [price]    Adds an item to a marketplace
  [forSale]
  marketplace_add_item_batch <marketplace> <csv>                Adds multiple items to a marketplace
  marketplace_remove_item <marketplace> <object>                Removes an item from a marketplace
  storefront_section_add_item <marketplace> <sku> [section]     Adds an item to a marketplace storefront section
  storefront_section_remove_item <marketplace> <sku>            Removes an item from a marketplace storefront section
  [writeToken]
  tenant_provision <tenant>                                     Provisions a new tenant account with standard media
                                                                libraries and content types. Note this account must be
                                                                created using space_tenant_create.
  tenant_add_consumer_group <tenant>                            Deploys a BaseTenantConsumerGroup and adds it to this
                                                                tenant's contract.
  tenant_get_minter_config <tenant> [options]                   Gets the minter configuration for this tenant key
  tenant_create_minter_config <tenant> [options]                Creates the minter configuration for this tenant key
  tenant_replace_minter_config <tenant> [options]               Creates the minter configuration for this tenant key
  tenant_deploy_helper_contracts <tenant> [options]             Deploys the minter helper and transfer proxy contracts
                                                                using the authority service as the minter. Specify
                                                                option proxy or minthelper to only deploy that specific
                                                                contract.
  tenant_delete_minter_config <tenant> [options]                Creates the minter configuration for this tenant key
  tenant_publish_data <tenant> <content_hash> [options]         Submits the new version hash of the tenant Fabric object
                                                                for validation. The top level Eluvio Live object link
                                                                will be updated if there are no errors.
  notif_send <user_addr> <tenant> <event>                       Sends a notification (using the notification service).
  notif_send_token_update <nft_addr> <tenant>                   Sends a TOKEN_UPDATED notification to all owners of this
                                                                NFT.
  account_create <funds> <account_name> <tenant_admins>         Create a new account -> mnemonic, address, private key
  account_show                                                  Shows current account information.
  payment_contract_create addresses shares                      Deploy a payment contract for revenue split.
  payment_contract_show addr token_addr                         Show status of payment contract stakeholders.
  payment_release addr token_addr                               Retrieve payment from payment splitter contract as a
                                                                payee or for a payee using --payee flag
  token_contract_create <name> <symbol> <decimals> <amount>     Deploy elv token  contract
  [options]
  token_add_minter <addr> <minter>                              Add minter or mint helper address to NFT or Token
  token_renounce_minter <addr>                                  Renounce the minter(msg.sender) from NFT or Token
  token_is_minter <addr> <minter>                               check if minter to NFT or Token
  content_set_policy <object> <policy_path> [data]              Set the policy on an existing content object. This also
                                                                sets the delegate on the object contract to itself.
  content_set_policy_delegate <object> <delegate>               Set the policy delegate on the object contract.
  content_get_policy <object>                                   Get the content object policy from the object metadata
                                                                and the delegate from the object's contract meta
  content_clear_policy <object>                                 Remove content object policy from the object metadata
                                                                and the delegate from the object's contract meta
  list [options]                                                List the Eluvio Live Tenants
  shuffle <file> [options]                                      Sort each line deterministically based on the seed
  admin_health [options]                                        Checks the health of the Authority Service APIs. Note
                                                                the current key must be a system admin configured in the
                                                                AuthD servers.

Options:
      --version  Show version number                                                                           [boolean]
  -v, --verbose  Verbose mode                                                                                  [boolean]
      --as_url   Alternate authority service URL (include '/as/' route if necessary)                            [string]
      --help     Show help                                                                                     [boolean]

Not enough non-option arguments: got 0, need at least 1
```

**Review by elv-stav — APPROVED** [review]

File: utilities/EluvioLiveCli.js [hunk 3]
```diff
@@ -824,7 +824,7 @@ const CmdAccountShow = async () => {
   console.log("Account Show\n");
 
   try {
-    await Init({ debugLogging: argv.verbose, asUrl: argv.as_url });
+    await Init({ debugLogging: false });
```
Comment (elv-Loren) [inline utilities/EluvioLiveCli.js:827]: did we intend to remove verbose and asUrl here?

Comment (elv-todd) [reply utilities/EluvioLiveCli.js:827]: yep

**Pull request merged**